### DISCUSSION TITLE
How to hide Dock icon to make the browser window completely invisible?

### DISCUSSION THREAD

**Discussion by CatStudioApp** [tauri-apps/wry#995] [Q&A] [answered]
```rs
let window = WindowBuilder::new()
    .with_title("Hello World")
    .with_visible(false)
```

seems not working because it only hides the window, but the Dock item on macOS.

**Comment by FabianLars** [comment, marked as answer]
as far as i know on macos this can only be controlled at the application level. There are 2 relevant apis here, [set_activation_policy](https://docs.rs/tao/latest/x86_64-apple-darwin/tao/event_loop/struct.EventLoop.html#method.set_activation_policy) and [hide_application](https://docs.rs/tao/latest/x86_64-apple-darwin/tao/event_loop/struct.EventLoopWindowTarget.html#method.hide_application).

**Reply by CatStudioApp** [reply]
Thank you!

Is there any way to do this on Linux/Windows as well?

**Reply by FabianLars** [reply]
On Linux/Windows hiding the window should be enough if i remember correctly.
